This patch is part of a series of patches for festival for
Debian GNU/Linux, to provide Debian specific building of festival

Index: b/config/config.in
===================================================================
--- a/config/config.in
+++ b/config/config.in
@@ -10,7 +10,7 @@
 ###########################################################################
 ## Which speech tools to use
 
-EST=$(TOP)/../speech_tools
+EST=/usr/lib/speech_tools
 
 ###########################################################################
 ## Where the festival tree will be installed.
@@ -20,7 +20,7 @@
 ## You may need to set this explicitly if automounter or NFS
 ## side effects cause problems
 
-FESTIVAL_HOME := $(shell (cd $(TOP); pwd))
+FESTIVAL_HOME := /usr/share/festival
 
 ###########################################################################
 ## Feature selection.
